pkg/api/core/v1/errors/errors.go

```diff
@@ -109,7 +109,8 @@ func NewBadRequestError(msg string) APIError {
 	return NewAPIError(msg, http.StatusBadRequest)
 }
 
-// NewBadRequestErrorf constructs an API error for general issues with a request, with a formatted message
+// NewBadRequestErrorf constructs an API error for general issues with a request, with a formatted
+// message
```

Comment:
Was this needed for the linter? If so I would just add the newline before the comma `,` (nicer imho)

```suggestion
// NewBadRequestErrorf constructs an API error for general issues with a request,
// with a formatted message
```

Reply:
No. Can undo them.

Reply:
Done. Commit 9b1343f364ddc7d955c97ac6dbee1d1ab412fd09